lib/temporal.js

```diff
@@ -228,15 +232,26 @@ function processQueue() {
 
 ["loop", "delay"].forEach(function(type) {
   exportable[type] = function(time, operation) {
+
+    var opts;
+
     if (typeof time === "function") {
       operation = time;
       time = 10;
     }
-    var task = new Task({
-      time: time,
-      type: type,
-      task: operation
-    });
+
+    if (typeof time === "object") {
+      opts = time;
+      opts.type = type;
+    } else {
+      opts = {
+        time: time,
+        type: type,
+        task: operation
```

Comment:
Let's rename `operation` to `task` in the parameters and then below change `var task = new Task(...)` to `var instance = new Task(...)`